### DISCUSSION TITLE
STB support?

### DISCUSSION THREAD

**Discussion by Odroid13** [4gray/iptvnator#303] [Ideas]
Hello, would it be possible to add support for the STB protocol? (Connection with MAC address)

There is no software for Linux, there is SFVIP Player but only on Windows, works with Wine but not too stable.

It would be wonderful to have this STB connection option.

THANKS

**Comment by 4gray** [comment]
Hi, yes, it will be available in the next version of the app.

**Comment by Odroid13** [comment]
Very good news, I look forward to this new version. Thank you very much for your work ^^

**Reply by 4gray** [reply]
STB support is there, please check v0.15

**Reply by Odroid13** [reply]
Thanks for support STB, but unfortunately it doesn't work.

**Comment by fabianr66** [comment]
I've seen STB options where you have URL, MAC and user/pass.... can you add this functionality?

Keep up the excellent work....
